File: JAVA/jeonggeun/beakjoon/N7510.java
package beakjun;

import java.util.Arrays;
import java.util.Scanner;

public class N7510 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int i = 0;
		int n = scanner.nextInt();
		
		while(i < n) {
			int[] len = new int[3];
			for(int j = 0; j < 3; j ++) {
				len[j] = scanner.nextInt();
			}
			Arrays.sort(len);
			
			System.out.println("Scenario #" + (i+1) + ":");

			if(Math.pow(len[2],2) == Math.pow(len[0], 2) + Math.pow(len[1],2)) {
				System.out.println("yes");
				System.out.println();
			}else {
				System.out.println("no");
				System.out.println();
			}
			i++;
		}
	}

}

File: Python/jeongguen/beakjoon/bronze/N7510.py
N = int(input())

for i in range(0,N):
    a,b,c = input().split()
    max = int(a)
    if max > int(b) and max > int(b):
        if pow(max,2) == pow(int(b),2) + pow(int(c),2):
            print("Scenario #%d:"%(i+1))
            print("yes")
            print()
        else:
            print("Scenario #%d:"%(i+1))
            print("no")
            print()           
    elif int(b) > int(c):
        if pow(int(b),2) == pow(max,2) + pow(int(c),2):
            print("Scenario #%d:"%(i+1))
            print("yes")
            print()
        else:
            print("Scenario #%d:"%(i+1))
            print("no")
            print()
    else:
        if pow(int(c),2) == pow(max,2) + pow(int(b),2):
            print("Scenario #%d:"%(i+1))
            print("yes")
            print()
        else:
            print("Scenario #%d:"%(i+1))
            print("no")   
            print()
         
